aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -497,142 +496,124 @@ impl AptosVM {
 
     fn finish_aborted_transaction(
         &self,
-        prologue_change_set: VMChangeSet,
+        prologue_session_change_set: SystemSessionChangeSet,
         gas_meter: &mut impl AptosGasMeter,
         txn_data: &TransactionMetadata,
         resolver: &impl AptosMoveResolver,
         status: ExecutionStatus,
+        txn_aux_data: TransactionAuxiliaryData,
         log_context: &AdapterLogSchema,
         change_set_configs: &ChangeSetConfigs,
         traversal_context: &mut TraversalContext,
-    ) -> Result<(VMChangeSet, FeeStatement, ExecutionStatus), VMStatus> {
-        // Storage refund is zero since no slots are deleted in aborted transactions.
```

Comment:
wondering - why inline 0? also this comment is then not present anywhere any more

Reply:
Oops, addressed comments too fast :) Bringing back.